step.yml

```diff
@@ -6,124 +6,125 @@ website: https://github.com/bitrise-steplib/steps-export-xcarchive
-        
-        If you select `auto-detect`, the step will figure out the proper export method  
-        based on the provisioning profile embedded into the generated xcode archive.
-      value_options:
-      - "auto-detect"
-      - "app-store"
-      - "ad-hoc"
-      - "enterprise"
-      - "development"
-      is_required: true
-  - upload_bitcode: "yes"
-    opts:
-      title: "Include bitcode"
-      summary: ""
-      description: |-
-        For __App Store__ exports, should the package include bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - compile_bitcode: "yes"
-    opts:
-      title: "Rebuild from bitcode"
-      summary: ""
-      description: |-
-        For __non-App Store__ exports, should Xcode re-compile the app from bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - team_id: 
-    opts:
-      title: "The Developer Portal team to use for this export"
-      summary: ""
-      description: |-
-        The Developer Portal team to use for this export.
+    The input value sets `xcodebuild`’s `-archivePath` option.
+    is_required: true
 
-        Format example:
+- product: "app"
+  opts:
+    title: "Select a product to distribute"
+    summary: ""
+    description: |-
+      Describes which product to export.
+    value_options:
+    - "app"
+    - "app-clip"
+    is_required: true
 
-        - `1MZX23ABCD4`
-  - product: "app"
-    opts:
-      title: "Select a product to distribute"
-      summary: ""
-      description: |-
-        Describes which product to export.   
-        Possible options are App or App Clip.
-      value_options:
-      - "app"
-      - "app-clip"
-      is_required: true
-  - custom_export_options_plist_content:
-    opts:
-      title: "Custom export options plist content"
-      description: |-
-        Specifies a custom export options plist content that configures archive exporting.
-        If empty, step generates these options based on the embedded provisioning profile,
-        with default values.
+- distribution_method:
+  opts:
+    title: "Distribution method"
+    summary: "Describes how Xcode should export the archive."
+    description: |-
+      Describes how Xcode should export the archive.
+    value_options:
+    - "development"
+    - "app-store"
+    - "ad-hoc"
+    - "enterprise"
+    is_required: true
 
-        Auto generated export options available for export methods:
+# IPA export configuration
 
-        - app-store
-        - ad-hoc
-        - enterprise
-        - development
+- export_development_team:
+  opts:
+    category: "IPA export configuration"
+    title: "Developer Portal team"
+    summary: "The Developer Portal team to use for this export"
+    description: |-
+      The Developer Portal team to use for this export.
 
-        If step doesn't find export method based on provisioning profile, development will be use.
+      Format example: `1MZX23ABCD4`
```

Comment:
The Developer Portal team to use for this export.

Defaults to the team used to build the archive.